src/control_devices/CCPACSControlSurfaceWingCutOut.cpp

```diff
@@ -132,22 +243,50 @@ CCPACSControlSurfaceWingCutOut::GetCutoutCS(bool isInnerBorder, const CCPACSCont
     double tXsi = outerShapeBorder->getXsiTE(); // this is always 1.0
 
     const auto& segment = ComponentSegment(*this);
-    gp_Pnt pLE = segment.GetPoint(lEta, lXsi);
-    gp_Pnt pTE = segment.GetPoint(tEta, tXsi);
+    gp_Pnt pLE          = segment.GetPoint(lEta, lXsi);
+    gp_Pnt pTE          = segment.GetPoint(tEta, tXsi);
 
     CTiglControlSurfaceBorderCoordinateSystem coords(pLE, pTE, upDir);
     return coords;
 }
 
-const CCPACSWingComponentSegment &ComponentSegment(const CCPACSControlSurfaceWingCutOut& self)
+CTiglControlSurfaceBorderCoordinateSystem CCPACSControlSurfaceWingCutOut::GetCutoutCS(
```

Comment:
I am sure we can get more DRYness here as well. The implementation starting in line 262 only needs a leading edge eta and a trailing edge eta, which can both be supplied from either a TED or a LED. Therefore at least that part of the code can be shared for both TEDs and LEDs.